src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline asset` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+
+import click
+
+from .._common import _handle_error
+
+
+@click.group(name="asset")
+@_handle_error
+def cli_asset():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_asset.command(name="snapshot")
+@click.option("--root-dir", help="The root directory to snapshot. ")
+@click.option("--manifest-out-dir", help="Path to directory for manifest output. ")
+@click.option(
+    "--r",
+    help="Flag to recursively snapshot subdirectories. ",
+    is_flag=True,
+    show_default=True,
+    default=False,
+)
+@_handle_error
+def asset_snapshot(**args):
+    """
+    Creates manifest of files specified root directory.
+    """
+    click.echo("snapshot taken")
+
+
+@cli_asset.command(name="upload")
+@click.option("--manifest", help="The path to manifest folder of directory specified for upload. ")
+@click.option("--farm-id", help="The AWS Deadline Cloud Farm to use. ")
+@click.option("--queue-id", help="The AWS Deadline Cloud Queue to use. ")
+@click.option(
+    "--update",
+    help="Flag to update manifest before upload. ",
+    is_flag=True,
+    show_default=True,
+    default=False,
+)
+@_handle_error
+def asset_upload(**args):
+    """
+    Uploads the assets in the provided manifest file to S3.
+    """
+    click.echo("upload done")
+
+
+@cli_asset.command(name="diff")
+@click.option("--root-dir", help="The root directory to compare changes to. ")
+@click.option("--manifest", help="The path to manifest of working directory to show changes of. ")
+@click.option(
+    "--print",
+    help="Pretty prints diff information. ",
+    is_flag=True,
+    show_default=True,
+    default=False,
+)
```

Comment:
Would this be similar to --v (verbosity) use case, or this is about formatting?

Reply:
This would be a formatting option:
Without --print, it would output changes in a manifest schema/raw json format, while using --print would show in a nicer formatted way, such as with git status

Reply:
In that case, would `--format` makes more sense? According to your reply, this CLI prints output changes regardless of the print flag.